lib/import.ml

```diff
@@ -13,12 +28,13 @@ let ( = ) = Int.equal
 let ( == ) = [ `Use_phys_equal ]
 let ( < ) (x : int) (y : int) = x < y
 let ( > ) (x : int) (y : int) = x > y
-let min = Int.min
-let max = Int.max
+let min (x : int) (y : int) = Stdlib.min x y
+let max (x : int) (y : int) = Stdlib.max x y
```

Comment:
These specialize back to integer comparison, right?

Reply:
I don't think OCaml has any specialized versions for `min` and `max` (as opposed to `compare`), but it should work the same regardless.
We can have this instead if you prefer:
```ocaml
let min (x : int) (y : int) = if x <= y then x else y
let max (x : int) (y : int) = if x >= y then x else y
```